detail/node.vue

```diff
@@ -42,30 +55,67 @@ export default {
   },
 
   data() {
+    const metricPoller = new Poler(this.loadMetrics, METRICS_POLL_RATE_MS, MAX_FAILURES);
+
+    metricPoller.start();
```

Comment:
In `mounted()`?, There's no point setting up the polling on a server-side render.

Reply:
moved